src/plugins/dna_export/src/ExportSequences2MSADialog.cpp

```diff
@@ -22,21 +22,18 @@
 #include "ExportSequences2MSADialog.h"
 
 #include <QMessageBox>
-#include <QPushButton>
 
 #include <U2Core/AppContext.h>
 #include <U2Core/BaseDocumentFormats.h>
 #include <U2Core/L10n.h>
-#include <U2Core/Settings.h>
 
-#include <U2Gui/DialogUtils.h>
 #include <U2Gui/HelpButton.h>
 #include <U2Gui/SaveDocumentController.h>
 
 namespace U2 {
 
 ExportSequences2MSADialog::ExportSequences2MSADialog(QWidget *p, const QString &defaultUrl)
-    : QDialog(p),
+    : QDialog(p), Ui_ExportSequences2MSADialog(),
```

Comment:
This isn't necessary, but doesn't make any problems

Reply:
OK, looks like I need to disable this warning/inspection in my CLion